src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,93 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    
+    private final static ArrayList<Tagging> ALL_TAGGING_CHANGES = new ArrayList<Tagging>();
+    
+    private final static String ADDED_PREFIX   = "+";
+    private final static String DELETED_PREFIX = "-";
+    
+    private final static String MESSAGE_TAGGING_CHANGES = "%1$s %2$s %3$s";
+    private final static String EXAMPLE_TAGGING_CHANGE  = "+ Jake Woo [friend]";
+    
+    private final static String LS = System.lineSeparator();
+    
+    private final Tag    tag;
+    private final Person person;
+    private boolean      isLive; // whether tagging is live or deleted
+    
+    /**
+     * Creates a tagging object and records this creation in
+     * {@link #ALL_TAGGING_CHANGES}
+     */
+    public Tagging(Person person, Tag tag) {
```

Comment:
is there a way to straight away create a record of tagging that is meant for deletion? (i.e. can we set isLive straight from the constructor?)